.github/workflows/pull-request.yml

```diff
@@ -168,7 +168,6 @@ jobs:
           path: '**/dist'
           key: dist-${{ env.rid }}
 
-
```

Comment:
This change isn't required right? Can we revert it

Reply:
Removed